sqlite.zig

```diff
@@ -1444,7 +1444,8 @@ pub fn Iterator(comptime Type: type) type {
                         }
                         @compileError("enum column " ++ @typeName(FieldType) ++ " must have a BaseType of either string or int");
                     },
-                    .Struct => {
+                    .Struct => |TI| {
+                        if (TI.layout == .Packed) return @bitCast(FieldType, try self.readInt(TI.backing_integer.?, i));
```

Comment:
This won't work since we've updated the zig version, @bitCast takes only one argument

Reply:
been stuck on an old version, updated